docker-compose.yml

```diff
@@ -8,9 +8,9 @@ services:
       - "46656-46657:46656-46657"
     environment:
       - ID=0
-      - LOG=${LOG:-tendermint.log}
+      - LOG=$${LOG:-tendermint.log}
     volumes:
-      - ${FOLDER:-./build}:/tendermint:Z
+      - ./build:/tendermint:Z
```

Comment:
@greg-szabo do we need these FOLDER vars?

Reply:
it threw a syntax error with the change as the recommended fix